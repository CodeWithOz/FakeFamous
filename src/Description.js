import React from 'react';

const Description = props => {
  return (
    <div>
      <h2 className="header">{props.name}</h2>
      <p className="description">{props.job}</p>
    </div>
  );
};

export default Description;
