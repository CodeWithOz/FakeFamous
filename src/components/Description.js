import React from 'react';

const Description = props => {
  return (
    <div>
      <h2 className="center aligned header">{props.name}</h2>
      <p className="center aligned description">{props.job}</p>
    </div>
  );
};

export default Description;
