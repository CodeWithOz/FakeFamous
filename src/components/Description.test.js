import React from 'react';
import Description from './Description';
import renderer from 'react-test-renderer';

test('Description component exists', () => {
  const description = renderer.create(
    <Description
      name="Firstname Lastname"
      job="Dummy job"
    />
  );

  // render the component
  const tree = description.toJSON();
  expect(tree).toMatchSnapshot();
});
