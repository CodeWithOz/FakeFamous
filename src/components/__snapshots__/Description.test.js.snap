// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description component exists 1`] = `
<div>
  <h2
    className="center aligned header"
  >
    Firstname Lastname
  </h2>
  <p
    className="center aligned description"
  >
    Dummy job
  </p>
</div>
`;
